api/REST-API.md

```diff
@@ -82,10 +81,8 @@ Complete the Links mentioned in the Materials sections. Follow step by step what
 
 ### Independent Practice
 
-In order to know whether you have successfully learned the lessons and materials, personal practice is most important. Do the following things:
- - Build a small application based on REST API.
- - The application can be anything, any idea that you think is useful.
-
+Go through the link and try to follow the tutorial:
```

Comment:
This would be perfect; however, it’s my fault for not telling you that they are actually going to be learning postgresQL and SQL first... is there a similar tutorial out there for node/express/postgresql/sql?

Reply:
Also, will  you remove the words “try to”?

Reply:
Do I need to add those links in prerequisites?

Reply:
Prerequisites are not the issue... at this point in the curriculum, they should be using postgresql and sql. Can you find a tutorial with those?

Reply:
Ya I have already found the tutorials. where should I put that?